src/functions-and-arrays.js

```diff
@@ -125,6 +260,6 @@ if (typeof module !== 'undefined') {
     uniquifyArray,
     doesWordExist,
     howManyTimes,
-    greatestProduct
+    greatestProduct,
   };
 }
```

Comment:
Very good solutions! Keep up the good work! 🙂

Reply:
Thanks a lot for the comments and the praise! 😄  I will update the solutions accordingly.